docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+    "\n",
+    "k = 5 \n",
+    "indicies = create_folds(lines, k=k) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "304653aa",
+   "metadata": {},
+   "source": [
+    "We train one model for each training/testing pair: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "6c877f7e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Model 0 already exists, skipping...\n",
+      "Model 1 already exists, skipping...\n",
+      "Model 2 already exists, skipping...\n",
+      "Model 3 already exists, skipping...\n",
+      "Model 4 already exists, skipping...\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(k): \n",
+    "    if os.path.exists('folds/fold%d/model/' % i): \n",
+    "        print('Model %d already exists, skipping...' % i) \n",
+    "    else: \n",
+    "        os.system(\n",
+    "            \"python3 run_ner.py --data_dir=folds/fold%d --bert_model=bert-base-cased \" \\\n",
+    "            \"--task_name=ner --output_dir=folds/fold%d/model --max_seq_length=256 \" \\\n",
+    "            \"--do_train --num_train_epochs 10 --warmup_proportion=0.1\" % (i, i)) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80621481",
+   "metadata": {},
+   "source": [
+    "## 4. Compute Out-of-Sample Predicted Probabilities \n",
+    "\n",
+    "We obtain the predicted probabilities for each sample using the model in which the sample was held out from training. Note that most tokenizers break down sentences into subword-level tokens, which are units smaller than word-level tokens. For example, the following sentence: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "560f8626",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Eu rejects German call to boycott British lamb.\n"
+     ]
+    }
+   ],
+   "source": [
+    "i = 0 \n",
+    "print(sentences[i]) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c97d0119",
+   "metadata": {},
+   "source": [
+    "is tokenized into: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "08446655",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['E', '##u', 'rejects', 'German', 'call', 'to', 'boycott', 'British', 'la', '##mb', '.']\n"
+     ]
+    }
+   ],
+   "source": [
+    "model = Ner(\"folds/fold0/model/\") \n",
+    "tokens = model.tokenize(sentences[i])[0] \n",
+    "print(tokens) \n",
+    "tokens = [token.replace('#', '') for token in tokens] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "493203d3",
+   "metadata": {},
+   "source": [
+    "`##` indicates that the token is a subword. We remove them and manually map them to the original tokens afterwards. We collect both the predictive probabilities and the tokens for each sentence. "
```

Comment:
"We remove them and manually map them to the original tokens afterwards"
-> 
"We will later remove these characters and then manually map these new tokens back to the original tokens, so that ..." 

Here ... should clarify why this is useful to do.